src/common_types.ts

```diff
@@ -282,12 +284,10 @@ export type SyntheticsConfig = {
 };
 
 /** Runner Payload types */
-export type JourneyResult = {
-  status: StatusValue;
-  error?: Error;
+export type JourneyResult = Partial<Journey> & {
   networkinfo?: PluginOutput['networkinfo'];
   browserconsole?: PluginOutput['browserconsole'];
-  steps?: Array<StepResult>;
+  stepsresults?: Array<StepResult>;
```

Comment:
what's the history behind keeping these lower case?

Reply:
Nothing I can think so, sort of kept it compatible with the Plugin types - https://github.com/elastic/synthetics/blob/03c74b85f8e76c103f2b55e981b275d6ac6fcd46/src/plugins/plugin-manager.ts#L36